src/containers/org/roles/OrgRoleContainer.js

```diff
@@ -51,18 +53,23 @@ const OrgRoleContainer = ({
   organizationId :UUID;
   roleId :UUID;
 |}) => {
-
+  const dispatch = useDispatch();
   const [selection, setSelection] = useState();
-
   const organization :?Organization = useSelector(selectOrganization(organizationId));
   const role :?Role = useMemo(() => (
     organization?.roles.find((orgRole) => orgRole.id === roleId)
   ), [organization, roleId]);
-
   const orgPath = useMemo(() => (
     Routes.ORG.replace(Routes.ORG_ID_PARAM, organizationId)
   ), [organizationId]);
 
+  useEffect(() => {
+    if (organization && !role) {
+      const membersPath = Routes.ORG_MEMBERS.replace(Routes.ORG_ID_PARAM, organizationId);
+      dispatch(goToRoute(membersPath));
+    }
```

Comment:
I think the condition should be if `REMOVE_ROLE_FROM_ORGANIZATION` is success, then route. It's possible for the id in the url to be incorrect, in which case `role` would not be defined, in which case it's probably better to show an error message.

Reply:
So should I not redirect and instead show an error message in the `OrgRoleContainer`? This flow is just weird with the deleting the role from this component.

Reply:
Then I could keep the modal open with a success or a fail?

Reply:
Once the role is deleted, the modal will close, and you will be rerouted to the members view.